tools.go

```diff
@@ -0,0 +1,13 @@
+// +build tools
+
+// Official workaround to track tool dependencies with go modules:
+// https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module
+
+package tools
+
+import (
```

Comment:
might be useful to separate based on how they are used?

Reply:
added comments to clarify